src/components/RollDiceButton.tsx

```diff
@@ -0,0 +1,169 @@
+import CloseIcon from '@mui/icons-material/Close';
+import {
+  Box,
+  Button,
+  FormControl,
+  Menu,
+  MenuItem,
+  Tooltip,
+  Typography,
+  styled,
+} from '@mui/material';
+import { grey } from '@mui/material/colors';
+import D4Icon from 'assets/images/dice/D4.svg';
+import D6Icon from 'assets/images/dice/D6.svg';
+import D8Icon from 'assets/images/dice/D8.svg';
+import D10Icon from 'assets/images/dice/D10.svg';
+import D12Icon from 'assets/images/dice/D12.svg';
+import D20Icon from 'assets/images/dice/D20.svg';
+import D100Icon from 'assets/images/dice/D100.svg';
+import React from 'react';
+
+import ParBox from './styled/ParBox';
+
+interface DiceMenuItem {
```

Comment:
Once the RollDiceButton has been moved to the components/charlist/ folder, create interfaces.ts and move it there.